src/ray/object_manager/test/ownership_object_directory_test.cc

```diff
@@ -130,10 +130,11 @@ class OwnershipBasedObjectDirectoryTest : public ::testing::Test {
               gcs_client_mock_,
               subscriber_.get(),
               &client_pool,
-              /*max_object_report_batch_size=*/20,
```

Comment:
I'd prefer to pass in `max_object_report_batch_size` so you can set it with different value in test instead of making it non-const.

Alternatively you can do something like 

```
class TaskEventBufferTest : public ::testing::Test {
 public:
  TaskEventBufferTest() {
    RayConfig::instance().initialize(
        R"(
{
  "task_events_report_interval_ms": 1000,
  "task_events_max_num_status_events_buffer_on_worker": 100,
  "task_events_send_batch_size": 100
}
  )");
```